### DISCUSSION TITLE
Image not building up

### DISCUSSION THREAD

**Discussion by algoexpert12** [apache/airflow#23331] [General]
### Apache Airflow version

2.2.3

### What happened

error: failed to solve: executor failed running [/bin/bash -o pipefail -o errexit -o nounset -o nolog -c pip install --no-cache-dir --user my-awesome-pip-dependency-to-add]: exit code: 1

### What you think should happen instead

Image should build successfully

### How to reproduce

./breeze --python 3.8 --backend sqlite

### Operating System

windows 11

### Versions of Apache Airflow Providers

2.2.3

### Deployment

Other

### Deployment details

breeze

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by tirkarthi** [comment]
cc: @potiuk

**Comment by potiuk** [comment]
This is likely propblem with your system (Windows). Looks like a number of people have it and seems to be related to clock synchronization (but I just searched Google which you could do as well so I recomment to try those things there and search more if it does not work.

https://askubuntu.com/questions/1059217/getting-release-is-not-valid-yet-while-updating-ubuntu-docker-container
